starknet_py/net/account/account.py

```diff
@@ -165,7 +165,8 @@ async def _get_resource_bounds(
 
             l1_resource_bounds = ResourceBounds(
                 max_amount=int(
-                    estimated_fee.gas_consumed * Account.ESTIMATED_AMOUNT_MULTIPLIER
+                    (estimated_fee.overall_fee / estimated_fee.gas_price)
```

Comment:
Should we account for `gas_price = 0` just in case? 

(0,0) resources can be used in `estimateFee` / `simulateTransactions`. It normally shouldn't happen, but I suppose better safe than sorry?